src/utils/helpers.ts

```diff
@@ -17,30 +17,38 @@ export const generateDownload = (
   URL.revokeObjectURL(url);
 };
 
-// TODO: Make filename timestamp.filetype
-export const downloadPDF = (content: string) => {
+const filename = () => {
```

Comment:
We should probably add a timestamp, something like this

```javascript
  const now = new Date();
  const year = now.getFullYear();
  const month = String(now.getMonth() + 1).padStart(2, '0');
  const day = String(now.getDate()).padStart(2, '0');
  const hour = String(now.getHours()).padStart(2, '0');
  const minute = String(now.getMinutes()).padStart(2, '0');
  const second = String(now.getSeconds()).padStart(2, '0');

  const timestamp = `${year}${month}${day}${hour}${minute}${second}`;
  
  ```

Reply:
Sure